### PR TITLE
Fix Psalm autoloading in Travis CI

### PR DESCRIPTION
**Pull request by paragonie-security**
We need to fix this before we tag `v2.0.19`.

### CODE REVIEW

File: other/phpunit-shim.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+class PHPUnit_Framework_TestCase extends \PHPUnit\Framework\TestCase
```
Comment (glensc) [inline other/phpunit-shim.php:3]: this is not needed if you use phpunit 4.8.36

it contains forward compatible `\PHPUnit\Framework\TestCase` class

```
    "phpunit/phpunit": "^4.8.36|5.*|6.*|7.*|8.*|9.*",
```

Comment (paragonie-security) [reply other/phpunit-shim.php:3]: The converse is the problem: Newer PHPUnit doesn't have the non-namespaced version.

Comment (paragonie-security) [reply other/phpunit-shim.php:3]: Keep in mind: We need tests to run on PHP 5.2.

**Comment by paragonie-security** [comment]
Current hypothesis: Old versions of packages have loose dependencies, which cause non-supported versions of PHP to have weird corner cases simply solvable by upgrading to supported versions of PHP.

Regardless, if you care that much about type-safety, the same code's being checked in every PHP version, so we only need the latest version of Psalm.
